src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -9,7 +9,24 @@
 package com.easypost.net;
 
 import com.easypost.EasyPost;
+import com.easypost.exception.Constants;
 import com.easypost.exception.EasyPostException;
+import com.easypost.exception.API.ForbiddenError;
+import com.easypost.exception.API.GatewayTimeoutError;
+import com.easypost.exception.API.InternalServerError;
+import com.easypost.exception.API.InvalidRequestError;
+import com.easypost.exception.API.MethodNotAllowedError;
+import com.easypost.exception.API.NotFoundError;
+import com.easypost.exception.API.PaymentError;
+import com.easypost.exception.API.RateLimitError;
+import com.easypost.exception.API.RedirectError;
+import com.easypost.exception.API.ServiceUnavailablError;
+import com.easypost.exception.API.TimeoutError;
+import com.easypost.exception.API.UnauthorizedError;
+import com.easypost.exception.API.UnknownApiError;
```

Comment:
Can we not just import `com.easypost.exception.API.*`?

Reply:
Yes, we could, but the linter will complain. The only issue with using wildcard import is it could clutter your local namespace. More details here: https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
I'm aware, this was the same question I asked in your interview 😉 

If the linter complains, then feel free to leave it. The reason why a wildcard error shouldn't matter much here is because we have complete control over the environment and don't really use other dependencies meaning the namespace collision possibility would be super tiny since we're only importing our own items.

Reply:
But it opens up others to potentially back-door accessing other classes (think a bad __init__ in Python)